src/gui/shell.cpp

```diff
@@ -502,7 +502,8 @@ void Shell::handleRedraw(const QByteArray& name, const QVariantList& opargs)
 			emit neovimSuspend();
 		}
 	} else if (name == "popupmenu_show") {
-		if (opargs.size() != 4
+		if (!(opargs.size() == 4
+          || (opargs.size() == 5 && opargs.at(4).canConvert<qint64>()))
```

Comment:
Just change it to `size() < 4` and add a comment about the 5th argument, that way we wont run into more issues like it.